packages/lesswrong/components/common/ForumDropdownMultiselect.tsx

```diff
@@ -66,6 +66,10 @@ const styles = (theme: ThemeType): JssStyles => ({
   menu: {
     marginTop: 28,
     ...(isFriendlyUI && {
+      "& .MuiPopover-paper": {
```

Comment:
These styles are copied from `DropdownMenu` - they make no difference in light mode, but IMO they look a _lot_ better in dark mode